docs/csharp/tutorials/nullable-reference-types.md

```diff
@@ -30,27 +30,16 @@ The code you'll write for this sample expresses that intent, and the compiler en
 
 ## Create the application and enable nullable reference types
 
-Create a new console application either in Visual Studio or from the command line using `dotnet new console`. Name the application `NullableIntroduction`. Once you've created the application, you'll need to enable C# 8 beta features. Open the `csproj` file, and add a `LangVersion` element to the `PropertyGroup` element:
+Create a new console application either in Visual Studio or from the command line using `dotnet new console`. Name the application `NullableIntroduction`. Once you've created the application, you'll need to enable C# 8 beta features. Open the `csproj` file, and add a `LangVersion` element to the `PropertyGroup` element. You must opt into the **nullable reference types** feature, even in C# 8 projects. That's because once the feature is turned on, existing reference variable declarations become **non-nullable reference types**. While that decision will help find issues where existing code may not have proper null-checks, it may not accurately reflect your original design intent. You turn on the feature by setting the `NullableContextOptions` element to `enable`:
 
 ```xml
 <LangVersion>8.0</LangVersion>
+<NullableContextOptions>enable</NullableContextOptions>
```

Comment:
Earlier you've mentioned VS 2019 preview 1 - this would require VS 2019 preview 2. It might be simplest just to require preview 2 rather than handle both.

Reply:
Good catch. I'll continue tracking only the latest preview.